avogadro/qtplugins/noncovalent/noncovalent.cpp

```diff
@@ -59,11 +70,7 @@ NonCovalent::NonCovalent(QObject *p) : ScenePlugin(p)
     Vector3ub(halogenBColor.red(), halogenBColor.green(), halogenBColor.blue()),
     Vector3ub(chalcogenBColor.red(), chalcogenBColor.green(), chalcogenBColor.blue())
   };
-  m_lineWidths = {
-    settings.value("nonCovalent/lineWidth0", 2.0).toFloat(),
-    settings.value("nonCovalent/lineWidth1", 2.0).toFloat(),
-    settings.value("nonCovalent/lineWidth2", 2.0).toFloat()
-  };
+  m_lineWidth = settings.value("nonCovalent/lineWidth0", 5.0).toFloat();
```

Comment:
Is it just one line width now?

Reply:
And this string doesn't match the string you use when saving the settings below: `lineWidth0` here vs. `lineWidth` below.

Reply:
Yes, sorry, I didn't notice that the render type doesn't support different line widths... Now that you mention it, I could just create three groups instead and allow individual line widths.